sycl/source/detail/ur.cpp

```diff
@@ -91,6 +91,13 @@ std::vector<PluginPtr> &initializeUr(ur_loader_config_handle_t LoaderConfig) {
   // std::call_once is blocking all other threads if a thread is already
   // creating a vector of plugins. So, no additional lock is needed.
   std::call_once(PluginsInitDone, [&]() {
+    // TODO: Remove this SYCL_PI_TRACE notification in the first patch release
```

Comment:
Note that this is a functional change for which we have no test so it will be easy to accidentally drop/break this diagnostics